core/src/main/java/com/amplifyframework/predictions/models/Cell.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.predictions.models;
+
+import androidx.annotation.NonNull;
+
+import java.util.Objects;
+
+/**
+ * A representation of table's individual cell.
```

Comment:
Hm wait. But why do we have tables for predictions? Is there a Table in the category types? How does this class related to the category behaviors?

Reply:
Based on the result data types, it seems that we support detecting a table from a document image. I put explanation for it in the `Table.java` class, but not sure how accurate that is because I haven't visited `identify` API yet.